package.json

```diff
@@ -72,6 +79,7 @@
     "react-test-renderer": "^16.14.0",
     "stylelint": "^13.6.1",
     "stylelint-config-sass-guidelines": "^7.0.0",
-    "stylelint-config-standard": "^20.0.0"
+    "stylelint-config-standard": "^20.0.0",
+    "typescript": "^4.1.3"
   }
 }
```

Comment:
I didn't do anything specific to try to undo it, since it's not easy to undo an `npm audit fix` when it's mixed in with actual changes to package.json. `npm audit fix` will generally only change the package-lock.json file to bump any packages to newer versions with security fixes, since the vast majority of packages will only be listed in package-lock.json.

Since everything seems to be building fine, I'm inclined to not touch the package-lock.json file any further and just merge this as is. Thanks for reminding me, though, since I had forgotten!